dev.gbstudio.gb-studio.yaml

```diff
@@ -0,0 +1,44 @@
+id: dev.gbstudio.gb-studio
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+command: gb-studio
+finish-args:
+  - --socket=fallback-x11
+  - --filesystem=host:rw
```

Comment:
why is it `host` permissions?

Reply:
Doesn't seem to use filesystem portal, so it's fine imho.